src/components/CreateInstructorPage/index.jsx

```diff
@@ -0,0 +1,169 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Helmet from 'react-helmet';
+
+import { CreateInstructorForm } from './CreateInstructorForm';
+import StatusAlert from '../StatusAlert';
+import LoadingSpinner from '../LoadingSpinner';
+
+class CreateInstructorPage extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      startedFetching: false,
+    };
+    this.handleInstructorCreate = this.handleInstructorCreate.bind(this);
+    this.setStartedFetching = this.setStartedFetching.bind(this);
+  }
+
+  componentDidMount() {
+    this.props.fetchInstructorOptions();
+    this.setStartedFetching();
+  }
+
+  setStartedFetching() {
+    this.setState({ startedFetching: true });
+  }
+
+  processSocialLinks(links) {
+    if (!links) {
+      return [];
+    }
+
+    return links.map(link => (
+      ({
+        id: null,
+        type: link.type,
+        title: link.title,
+        url: link.url,
+      })
+    ));
+  }
+
+  processAreasOfExpertise(areasOfExpertise) {
+    if (!areasOfExpertise) {
+      return [];
+    }
+
+    return areasOfExpertise.map(area => ({ id: null, value: area.value }));
+  }
+
+  processPosition(title, organization) {
+    return {
+      title,
+      organization,
+      organization_override: null,
+    };
+  }
+
+  handleInstructorCreate(options) {
```

Comment:
`options` feels like a weird name for this. I suspect this is some cargo-culting that you are yourself also cargo-culting way back to some OPTIONS fetch handler.